### PR TITLE
feat(website): MongoDB blog post

### PR DESCRIPTION
**Pull request by disq**
https://cloudquery-fibvbv0ym-cloudquery.vercel.app/blog/building-an-infrastructure-data-lake-with-mongodb

### CODE REVIEW

**Review by hermanschaaf — APPROVED** [review]
I like it, nice one! Just a small nit around using "platform" instead of "lake"

**Comment by erezrokah** [comment]
Can we merge this PR?

**Comment by disq** [comment]
> Can we merge this PR?

it's ready.

File: website/pages/blog/building-an-infrastructure-data-lake-with-mongodb.mdx [hunk 1]
```diff
@@ -0,0 +1,110 @@
+---
+title: Building an Infrastructure Data Lake with MongoDB
+tag: tutorial
+date: 2023/03/21
```
Comment (erezrokah) [inline website/pages/blog/building-an-infrastructure-data-lake-with-mongodb.mdx:4]:
```suggestion
date: 2023/03/27
```

Maybe we can just do this and merge?

**Pull request merged**